src/main/java/org/craftercms/studio/impl/v2/repository/blob/StudioBlobStoreResolverImpl.java

```diff
@@ -168,7 +172,7 @@ public boolean isBlob(String site, String path) throws ServiceLayerException {
         try {
             return getByPaths(site, path) != null;
         } catch (BlobStoreConfigurationMissingException e) {
-            logger.debug("Blob store configuration is missing or invalid");
+            logger.debug("The blob store configuration is missing or invalid in site '{}'", site, e);
```

Comment:
Discuss with AV and JM. Is this the right log level and should the exception be swallowed?